Lib/test/support/__init__.py

```diff
@@ -2978,7 +2978,7 @@ def fd_count():
     if sys.platform.startswith(('linux', 'freebsd')):
         try:
             names = os.listdir("/proc/self/fd")
-            # Substract one because listdir() opens internally a file
+            # Subtract one because listdir() opens internally a file
```

Comment:
```suggestion
            # Subtract one because listdir() internally opens a file
```

Reply:
@minho42 Even though I could edit the file and commit the same change, I cannot commit 'suggestions' to your PR.  Please see if you can.